index.js

```diff
@@ -0,0 +1,132 @@
+
+const arr = [];
```

Comment:
taka nazwa nic nie mówi, co to za `arr`. to że to jest pusta tablica zresztą to każdy widzie :) to powinno się jakoś nazywać: `inputValues` czy czymkolwiek ta tablica jest